packages/typespec-azure-resource-manager/lib/common-types/types.tsp

```diff
@@ -11,24 +12,34 @@ model ArmResourceBase {}
  * Base model that defines common properties for all Azure Resource Manager resources.
  */
 @doc("Common properties for all Azure Resource Manager resources.")
-model ArmResource extends ArmResourceBase {
+model Resource extends ArmResourceBase {
```

Comment:
Do we still need ArmResourceBase?  I think we may have this as a template parameter constraint, but I wonder if we can't just use Resource instead.

Reply:
Agreed.